src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -292,10 +293,20 @@ public IActionResult GetAdminToken()
         [HttpGet]
         [HttpPost]
         [Route("admin/warmup")]
-        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
         [RequiresRunningHost]
         public async Task<IActionResult> Warmup([FromServices] WebJobsScriptHostService hostService)
         {
+            // Endpoint only for Windows Elastic Premium or Linux App Service plans
+            if (!(_environment.IsLinuxAppService() || _environment.IsWindowsElasticPremium()))
+            {
+                return NotFound();
```

Comment:
Should this be a 400? That's what I did in the GetScaleStatus endpoint when called incorrectly - https://github.com/Azure/azure-functions-host/blob/3ceb9efd6842a8d2a38002e390463c1f2dca1f80/src/WebJobs.Script.WebHost/Controllers/HostController.cs#L140.  We do the same in some other APIs

Reply:
Fair. I think the idea is that this API is non-existent for other skus.
I guess we would eventually return 401s for those skus in future. So, 400 is probably a good idea for now.